google/cloud/bigquery/schema.py

```diff
@@ -549,33 +549,29 @@ def _build_schema_resource(fields):
 
 
 def _to_schema_fields(schema):
-    """Coerce `schema` to a list of schema field instances.
-
-    Args:
-        schema(Sequence[Union[ \
-            :class:`~google.cloud.bigquery.schema.SchemaField`, \
-            Mapping[str, Any] \
-        ]]):
-            Table schema to convert. If some items are passed as mappings,
-            their content must be compatible with
-            :meth:`~google.cloud.bigquery.schema.SchemaField.from_api_repr`.
-
-    Returns:
-        Sequence[:class:`~google.cloud.bigquery.schema.SchemaField`]
-
-    Raises:
-        Exception: If ``schema`` is not a sequence, or if any item in the
-        sequence is not a :class:`~google.cloud.bigquery.schema.SchemaField`
-        instance or a compatible mapping representation of the field.
+    """TODO docstring
```

Comment:
Flagging with a comment just so we don't lose track of this TODO.

Reply:
Done.